packages/editor/src/panels/files/fileitem.tsx

```diff
@@ -114,7 +114,7 @@ function TableView({ file, onClick, onDoubleClick, isSelected, drag, drop, isOve
     staticResourceModifiedDates.set(modifiedDates)
   }, [staticResourceData.status])
 
-  const thumbnailURL = file.thumbnailURL
+  const thumbnailURL = file?.thumbnailURL
```

Comment:
Hey Daniel, can you explain the case when `file` is undefined?

Reply:
got it. I think some files are undefined because the file browser is not refreshed after a file is deleted.

Reply:
Sorry I totally missed this last week. Do you think the core issue is fixable. It seemed like a race condition most of the time. I couldn't actually reproduce the bug from the ticket. But I know this will fix it.